hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -1208,6 +1212,9 @@ public void addImplicitCostsIn(final TxnAccessor accessor) {
 		public void run(
 				final MessageFrame frame
 		) {
+			transferSemanticChecks.fullPureValidation(transactionBody.getCryptoTransfer().getTransfers(),
```

Comment:
This line shouldn't be here, we may want to re-merge `eth-tx-interop` in `native-staking` and then `native-staking` into this branch to be sure we're not adding it back in. 😞